Makefile

```diff
@@ -11,6 +11,7 @@ pidfile /tmp/redis1.pid
 logfile /tmp/redis1.log
 save ""
 appendonly no
+enable-module-command yes
```

Comment:
hi, @sazzad16 `redis.conf` does not seem to support this configuration

Reply:
@yangbodong22011 This is merged in unstable just yesterday. Perhaps they're still working on it. But tests in our PRs are failing. So we have to react.

Reply:
ohh，thanks, i noticed it : https://github.com/redis/redis/pull/9920